bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/GarbageCollectorThread.java

```diff
@@ -489,6 +505,17 @@ public void runWithFlags(boolean force, boolean suspendMajor, boolean suspendMin
                     minorCompacting.set(false);
                 }
             }
+            if (enableEntryLocationCompaction && (curTime - lastEntryLocationCompactionTime
```

Comment:
We'd better introduce a random factor for it. If we roll restart the BookKeeper cluster, and all the bookies start time are the same, which will make all the bookies triggered RocksDB compaction at the same time and impact the read latency.

Reply:
The random factor can be any time in [0, entryLocationCompactionInterval]